src/controllers/overseas.entity.due.diligence.controller.ts

```diff
@@ -1,12 +1,12 @@
 import { NextFunction, Request, Response } from "express";
 
 import * as config from "../config";
-import { getDueDiligence, postDueDiligence } from "../utils/due.diligence";
+import { getDueDiligence, postDueDiligence } from "../utils/due.diligence.overseas.entity";
 
 export const get = (req: Request, res: Response, next: NextFunction) => {
-  getDueDiligence(req, res, next, config.WHO_IS_MAKING_FILING_URL, config.OVERSEAS_ENTITY_DUE_DILIGENCE_PAGE);
+  getDueDiligence(req, res, next, config.OVERSEAS_ENTITY_DUE_DILIGENCE_PAGE, config.WHO_IS_MAKING_FILING_URL);
 };
 
 export const post = (req: Request, res: Response, next: NextFunction) => {
-  postDueDiligence(req, res, next, config.ENTITY_URL, true);
+  postDueDiligence(req, res, next, config.ENTITY_URL, false);
```

Comment:
Is the last arg isRegistration: boolean? If so then should it not be true as here it is registration journey?

Reply:
Good spot! I think I got confused with the names of the due diligence controllers and put in the wrong condition.